lapis-e2e/test/alignedNucleotideSequence.spec.ts

```diff
@@ -14,8 +13,11 @@ describe('The /alignedNucleotideSequence endpoint', () => {
     });
 
     expect(result).to.have.length(100);
-    expect(result[0].primaryKey).to.equal('key_3259931');
-    expect(result[0].main).to.have.length(29903);
+    const match = result.find(
+      (item: { primaryKey: string; main: string | any[] }) =>
+        item.primaryKey === 'key_3259931' && item.main?.length === 29903
+    );
+    expect(match).to.exist;
```

Comment:
This only yields "AssertionError: expected undefined to exist" when it fails -> I would prefer to `orderBy` the result instead. The tests will be simpler, more stable und yield better error messages.